dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -411,6 +411,9 @@ referral.desc=Specify the behavior for LDAP referrals. The default behavior is t
 referral.ignore=Ignore LDAP referrals.
 referral.follow=Follow LDAP referrals.
 
+primaryServerQueryTimeInterval=Primary server query time interval
+primaryServerQueryTimeInterval.desc=Polling interval for testing the primary server availability(in minutes)
```

Comment:
Change
Polling interval for testing the primary server availability(in minutes)
To
The polling interval, in minutes, at which the primary server is tested for availability.

Is it possible to say what component or process is testing the server for availability? For example

The polling interval, in minutes, at which (X) tests the primary server for availability.